src/pal/src/loader/module.cpp

```diff
@@ -1628,6 +1628,12 @@ static HMODULE LOADLoadLibrary(LPCSTR shortAsciiName, BOOL fDynamic)
     LockModuleList();
 
     dl_handle = LOADLoadLibraryDirect(shortAsciiName);
+#ifdef __APPLE__
+    if (!dl_handle)
```

Comment:
Ah you are right, I forgot that this method is *not* just loading libc. Will fix. :)